documentation/ipi-install/modules/ipi-install-creating-an-rhcos-images-cache.adoc

```diff
@@ -108,3 +108,5 @@ Use the following steps to install a container that contains the images.
 -p 8080:8080/tcp \
 registry.centos.org/centos/httpd-24-centos7:latest
 ----
+
+. The above command creates a caching webserver with the name `rhcos_image_cache` which will be serving the images for deployment. The first image `${RHCOS_PATH}${RHCOS_QEMU_URI}?sha256=${RHCOS_QEMU_SHA_UNCOMPRESSED}` will be used as `bootstrapOSImage` and the second image `${RHCOS_PATH}${RHCOS_OPENSTACK_URI}?sha256=${RHCOS_OPENSTACK_SHA_COMPRESSED}` will be used as `clusterOSImage` in `install-config.yaml` file as shown in xref:additional-install-config-parameters_{context}[Additional install-config parameters] section. 
```

Comment:
. The above command creates a caching webserver with the name `rhcos_image_cache` that serves the images for deployment. In order to take advantage of these images being served by the webserver, you must include them in the xref:additional-install-config-parameters_{context}[`install-config.yaml`] file. The first image `${RHCOS_PATH}${RHCOS_QEMU_URI}?sha256=${RHCOS_QEMU_SHA_UNCOMPRESSED}` will be used as `bootstrapOSImage`. The second image `${RHCOS_PATH}${RHCOS_OPENSTACK_URI}?sha256=${RHCOS_OPENSTACK_SHA_COMPRESSED}` will be used as `clusterOSImage`. 


--- 
@DirectedSoul1 what do you think of this?